### PR TITLE
Update installing-addons.md

### PR DESCRIPTION
**Pull request by maurodibert**
typo

### CODE REVIEW

**Comment by locks** [comment]
The current text seems fine to me. cc @acorncom

**Comment by maurodibert** [comment]
It's a double infinitive

File: guides/v3.3.0/tutorial/installing-addons.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Our update to `mirage/config.js` configures Mirage so that whenever Ember Data m
 We also specified a `namespace` of `/api` in our mirage configuration.
 Without this change, navigation to `/rentals` in our application would conflict with Mirage.
 
-In order for this to work, we need our application to default to making requests to the namespace of `/api`.
+In order for this to work, we need our application to default making requests to the namespace of `/api`.
```
Comment (jenweber) [inline guides/v3.3.0/tutorial/installing-addons.md]:
```suggestion
In order for this to work, we need to tell the application to send its requests to the `/api` namespace by default.
```

Comment (maurodibert) [reply guides/v3.3.0/tutorial/installing-addons.md]: Aaaaaawesome! ;-)

**Pull request merged**